topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,481 @@
+- Correction methods help control this risk to ensure that the results are reliable and that significant findings are not due to random chance.
+- This is done by computing the q-value, which is a measure of how many false positives are expected among the significant results. 
+- Options:
+      1. <u>Benjamini & Hochberg(BH)(aka false discovery rate(fdr))</u>: A common method used for FDR correction. It ranks the p-values from smallest to largest and adjusts them based on their rank and the total number of tests.
+      2. <u>Benjamini & Yekutieli(BY)</u>: Similar to Benjamini-Hochberg but includes a correction factor that accounts for the correlation between tests.
+      3. <u> Bonferroni correction</u>: Divides the significance threshold (alpha level) by the number of tests performed and then compares each p-value to this adjusted significance level to determine if it is statistically significant. 
+      4. <u>Hochberg </u>:  It is similar to the Bonferroni correction but is often more powerful, meaning it has better statistical power to detect true effects while controlling for false positives.
+      5. <u>Hommel</u>: controls the Family-Wise Error Rate (FWER) by adjusting p-values in a step-down fashion, starting from the smallest p-value and progressively increasing the threshold. It is more powerful than Bonferroni while maintaining strict error control.
+      6. <u>Holm</u>:  controls the Family-Wise Error Rate (FWER) by sequentially adjusting p-values from smallest to largest, and then comparing them to adjusted critical value, given by $$ \frac{α}{m−i+1} $$​, where α is the significance level (e.g., 0.05), m is the total number of hypothesis, and i is the rank of the p-value in the ordered list.\
+The corresponding null hypothesis is rejected for p-values less than the adjusted critical value and the process stops as soon as p-values equals or exceeds the corresponding critical value. This stepwise approach is less conservative than the Bonferroni correction, offering greater statistical power.\
+**FWER** is the probability of finding at least one false positive among all the tests performed, assuming all null hypotheses are true.\
+**FWER Control** is used to minimize the risk of incorrectly claiming significant results when there are none, thus maintaining the overall reliability of the results.\
+For more information on correction methods, [click here](https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/p.adjust).
+
+11.  **standardize**: Apply z-score so continuous metadata are on the same scale [ Default: TRUE ]
+12.  **plot_heatmap**: Generate a heatmap for the significant associations [ Default: TRUE ]
+13. **heatmap_first_n**: In heatmap, plot top N features with significant associations [ Default: 50 ]
+14.  **plot_scatter**: Generate scatter plots for the significant associations [ Default: TRUE ]
+15.   **cores**: The number of R processes to run in parallel [ Default: 1 ]
+
+# Model types in MaAslin2
```

Comment:
Not clear to me what this paragraph means, you already explain the models **analysis_method** - so its not clear what point 2 and 3 mean, can you choose these options ? Would remove or rephrase.

Reply:
These are not the option that can be chosen. These are broader categories of models under which the analysis methods explained above fall . For instance CPLM, ZICP etc come under generalized linear model. If user chooses random effect , then model works as linear mixed model.

Reply:
@paulzierep  I can make it more clear in the tutorial if you want. Let me know if the above explaination makes sense to you. I can add this explaination in the analysis method section  and remove the following para on these models.

Reply:
then you should say, that those are models chosen by maaslin depending on the data input and the parameter choice. Otherwise, it's confusing. However, if CPLM, ZICP lead to Generalized Linear Models then this should also be stated. You got the *When to Use:* part - but users cannot choose anything - so that would confuse me

Reply:
Understood. I  will make the necessary changes. Thanks for the detailed explanation :)